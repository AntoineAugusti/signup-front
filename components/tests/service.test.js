import ReactTestRenderer from 'react-test-renderer'
import DGFIP_DATA_SET from '../../mock/data/dgfip'
import Services from '../services'

describe('components | Services', () => {
  describe('render', () => {
    const props = {
      lists: DGFIP_DATA_SET.availables_data_sets[0].services
    }
    const renderer = ReactTestRenderer.create(<Services {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
