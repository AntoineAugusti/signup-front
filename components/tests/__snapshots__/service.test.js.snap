// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Services render should match snapshot 1`] = `
<div
  className="form__group"
>
  <label>
    Qui utilise ces donn√©es ?
  </label>
  <ul
    className="tag-list"
  >
    <li
      className="tag"
    >
      <a
        href="./static/images/services-providers/mes-aides-logo.png"
      >
        Mes aides
      </a>
    </li>
    <li
      className="tag"
    >
      <a
        href="./static/images/services-providers/anah-logo.png"
      >
        Anah
      </a>
    </li>
  </ul>
</div>
`;
